llvm/test/Transforms/LoopVectorize/AArch64/alias_mask.ll

```diff
@@ -0,0 +1,884 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
```

Comment:
This test can be removed now? You could have some loop-tests if needed in llvm/test/CodeGen/AArch64/whilewr.ll, but the tests in llvm/test/Transforms/LoopVectorize should be for the vectorizer.

Reply:
Yeah I've moved these to whilewr.ll .